templates/global/seller-registration-form.php

```diff
@@ -30,9 +30,19 @@
         <strong id="url-alart-mgs" class="pull-right"></strong>
         <input type="text" class="input-text form-control" name="shopurl" id="seller-url" value="<?php echo ! empty( $data['shopurl'] ) ? esc_attr( $data['shopurl'] ) : ''; ?>" required="required" />
         <small><?php echo esc_url( home_url() . '/' . dokan_get_option( 'custom_store_url', 'dokan_general', 'store' ) ); ?>/<strong id="url-alart"></strong></small>
-    </p>
+    </p>                                   
 
     <?php
+
+    /**
+     * Store Address Fields
+     */
+    $is_address_enabled = dokan_get_option( 'enabled_address_on_reg', 'dokan_general', 'on' );
+
+    if ( $is_address_enabled == 'on' ) {
```

Comment:
directly use `if ( 'on' === dokan_get_option( 'enabled_address_on_reg', 'dokan_general', 'on' ); ) `

Reply:
Also default value should be `off` as you set `off` as default value on corresponding settings